.circleci/config.yml

```diff
@@ -102,10 +102,23 @@ workflows:
           context: sonarcloud
           requires:
             - test
+      - automated-test/run-sdk-tests:
```

Comment:
We need to restrict this to master branch only

Reply:
done as well